File: sdk/javascript-closure-angular/API/Client/ImpersonateTokenRequest.js
goog.provide('API.Client.ImpersonateTokenRequest');

/**
 * @record
 */
API.Client.ImpersonateTokenRequest = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.ImpersonateTokenRequest.prototype.clientID;

/**
 * @type {!Array<!API.Client.ShipmentItem>}
 * @export
 */
API.Client.ImpersonateTokenRequest.prototype.claims;

File: sdk/dart/lib/model/impersonate_token_request.dart
part of swagger.api;


@Entity()
class ImpersonateTokenRequest {
  
  @Property(name: 'ClientID')
  String clientID = null;
  

  @Property(name: 'Claims')
  List<ShipmentItem> claims = [];
  
  ImpersonateTokenRequest();

  @override
  String toString()  {
    return 'ImpersonateTokenRequest[clientID=$clientID, claims=$claims, ]';
  }

}

